rosie/chamber_of_deputies/tests/test_adapter.py

```diff
@@ -27,6 +30,18 @@ def setUp(self):
     def tearDown(self):
         shutil.rmtree(self.temp_path)
 
+    def test_quantity_of_downloaded_datasets_all_years_fetch(self):
+        years = AVAILABLE_YEARS
```

Comment:
Why do we need this? You use `years` in line 35 but go back to `AVAILABLE_YEARS` in 37.

Reply:
didn't noticed, thanks /o/